common/containers/UserForm/index.jsx

```diff
@@ -56,10 +56,18 @@ function handleSubmit(dispatch) {
 
 function mapStateToProps(state, props) {
   const {identifier} = props.params
-  const {app, users} = state
+  const {app, users, phases} = state
```

Comment:
Now that you're pulling data from the phases store, need to also be sure to retrieve the data from the backend! :)

Should add the following import:
```js
import {findPhases} from 'src/common/actions/phase'
```

And the following line to `fetchData()`:
```js
dispatch(findPhases())
```

This will automatically take care of (A) retrieving the data and (B) storing it in the redux store (which ultimately makes it available in `state`).